assets/javascripts/admin_groups.js

```diff
@@ -0,0 +1,24 @@
+function createGroup(form) {
+    if(!form.name.value.length) {
+        window.alert('Group name mustn\'t be empty!');
```

Comment:
alert is for debugging, jquery and bootstrap have so much more powerful for form validation